src/components/textbox/textbox.component.js

```diff
@@ -67,6 +68,7 @@ const Textbox = ({
   enforceCharacterLimit = true,
   characterLimit,
   warnOverLimit = false,
+  newValidationDesign,
```

Comment:
This should come from a context provider so consumers can opt-in on a per-application basis. We shouldn't support opting in on a per-component basis.

```jsx
<ValidationRedesignOptIn><App/></ValidationRedesignOptIn>
```